help.go

```diff
@@ -214,7 +215,7 @@ func printFlagSuggestions(lastArg string, flags []Flag, writer io.Writer) {
 			continue
 		}
 		// match if last argument matches this flag and it is not repeated
-		if strings.HasPrefix(name, cur) && cur != name && !cliArgContains(name, os.Args) {
+		if strings.HasPrefix(name, currentArg) && currentArg != name && !cliArgContains(name, os.Args) {
```

Comment:
After the prefix is stripped, it is not arg anymore, but a search string for a match. I would name it `match` or `lookup` or something line that.

Reply:
I just reverted back to `cur`, don't want to spend time on this.